Src/IronPython/Runtime/PythonAsciiEncoding.cs

```diff
@@ -193,7 +193,7 @@ class NonStrictEncoderFallbackBuffer : EncoderFallbackBuffer {
         private int _index;
 
         public override bool Fallback(char charUnknownHigh, char charUnknownLow, int index) {
-            throw PythonOps.UnicodeEncodeError("'ascii' codec can't encode character '\\u{0:X}{1:04X}' in position {1}", (int)charUnknownHigh, (int)charUnknownLow, index);
+            throw PythonOps.UnicodeEncodeError("'ascii' codec can't encode character '\\u{0:X}{1:04X}' in position {1}: ordinal not in range(128)", (int)charUnknownHigh, (int)charUnknownLow, index);
         }
 
         public override bool Fallback(char charUnknown, int index) {
```

Comment:
Ah, in the email I got it didn't show that it was for the second one, it just showed the first one.